cfy_manager/exceptions.py

```diff
@@ -48,3 +48,7 @@ class ClusteringError(BootstrapError):
 class RabbitNodeListError(StandardError):
     """Raised when there is an error listing rabbit nodes."""
     pass
+
+
+class DBNodeListError(StandardError):
```

Comment:
this can also just be `Exception` rather than `StandardError` (which is removed in py3, but it's not deprecated in 2.7 yet)

Reply:
Changed.